scripts/include/lib-translations.inc.php

```diff
@@ -87,8 +87,13 @@ function is_translatable ($filename) {
 	if (in_array(basename($filename), $files_not_translated)) {
 		return false;
 	}
+
+	// exclude filenames like htmlentities.xml
+	if (preg_match("/^entities\./", basename($filename))) {
+		return false;
+	}
 	
-	$files_matches = array('/internals/', '/internals2/', 'entities.');
+	$files_matches = array('/internals/', '/internals2/');
```

Comment:
At this point we don't have these folders any more, so this can be removed completely.